api/apps/restful_apis/dataset_api.py

```
@@ -556,6 +556,295 @@ async def get_knowledge_graph(tenant_id, dataset_id):
+        success, result = await dataset_api_service.clear_artifacts(
+            dataset_id, tenant_id,
+        )
+        if success:
+            return get_result(data=result)
+        return get_result(data=False, message=result, code=RetCode.AUTHENTICATION_ERROR)
+    except Exception as e:
+        logging.exception(e)
+        return get_error_data_result(message="Internal server error")
+
+
+@manager.route(
```

Ruff (0.15.20)

[error] Undefined name `manager`

(F821)